funnel/transports/sms.py

```diff
@@ -38,36 +94,37 @@ def send_via_exotel(phone_number, message):
         raise TransportConnectionError("Exotel not reachable")
 
 
-def send_via_twilio(phone_number, message):
-    sid = app.config['SMS_TWILIO_SID']
-    token = app.config['SMS_TWILIO_TOKEN']
-    try:
-        r = requests.post(
-            'https://api.twilio.com/2010-04-01/Accounts/{sid}/Messages.json'.format(
-                sid=sid
-            ),
-            auth=(sid, token),
-            data={
-                'From': app.config['SMS_TWILIO_FROM'],
-                'To': phone_number,
-                'Body': message,
-            },
-        )
-        if r.status_code in (200, 201):
-            # All good
-            jsonresponse = r.json()
-            return jsonresponse.get('sid')
-        raise TransportTransactionError("Twilio API error", r.status_code, r.text)
-    except requests.ConnectionError:
-        raise TransportConnectionError("Twilio not reachable")
+def _route_via_twilio(phone: str, message: str, callback: bool = False) -> str:
```

Comment:
Callback is necessary for tracking deliverability, so it shouldn't be default False